docs/releases/v2020.1.2.rst

```diff
@@ -0,0 +1,76 @@
+Gluon v2020.1.2
+===============
+
+Removed hardware support
+------------------------
+
+lantiq-xway
+~~~~~~~~~~~
+
+- AVM FRITZ!Box 7320 [#switchports_not_working]_
+- AVM FRITZ!Box 7330 [#switchports_not_working]_
+- AVM FRITZ!Box 7330 SL [#switchports_not_working]_
+
+.. [#switchports_not_working]
+  The switchports on these devices are not working properly (`#1943 <https://github.com/freifunk-gluon/gluon/issues/1943>`_)
+
+Bugfixes
+--------
+
+- Fixes a bug in the tunneldigger watchdog where the watchdog would incorrectly find itself while looking up the running tunneldigger process. It then went on and assumed a PID mismatch between the tunneldigger service and its PID file and therefore caused an unnecessary restart of the tunnel. (`#1952 <https://github.com/freifunk-gluon/gluon/issues/1952>`_)
+
+- Fixes an oversight in the firewalling of the respondd service where queries from the ``extra_prefix6`` would be dropped. (`#1941 <https://github.com/freifunk-gluon/gluon/issues/1941>`_)
```

Comment:
extra_prefixes6

Reply:
by the way, why isn't this mentioned in the site.rst doc...

Reply:
> extra_prefixes6

Done.

> by the way, why isn't this mentioned in the site.rst doc...

PR welcome.